ReSharper.FSharp/src/FSharp.Psi/src/Impl/GeneratedMembersUtil.cs

```diff
@@ -104,5 +105,27 @@ public static IEnumerable<IDeclaredElement> GetGeneratedMembers([NotNull] this I
 
       return result;
     }
+
+    public static IList<ITypeMember> GetGeneratedMembers(this IFSharpProperty property)
```

Comment:
This name seems wrong. It only fetches declared accessors, nothing that is additionally generated.